config/routes.rb

```diff
@@ -59,6 +59,10 @@
       resources :documents, only: [:create]
     end
 
+    get 'intent_to_file', to: 'intent_to_files#index'
+    get 'intent_to_file/compensation/active', to: 'intent_to_files#active_compensation'
+    post 'intent_to_file/compensation', to: 'intent_to_files#submit_compensation'
```

Comment:
My understanding is that an `intent_to_file` "resource", has the following attributes:
```
- id
- creation_date
- expiration_date
- participant_id
- source
- status
- type
```
Of those attributes, the only ones we allow a client of `vets-api` to control upon creation is `type`.

Might it make more sense for the route to be:

- `POST /intent_to_file/{type}`

to allow for future creation of different ITF types?

As it stands now, `compensation` is the only valid value here, so the static route of `POST /intent_to_file/compensation` is effectively equivalent and not a blocker for this PR.

Reply:
As @omgitsbillryan mentions `/intent_to_file/compensation` and `/intent_to_file/{type}` are functionally the same but as the EVSS endpoint is `/intenttofile/v1/{type}` it's probably best to match that.

Reply:
@kreek @omgitsbillryan I've changed the routing around to using a dynamic `type` in the path.

Reply:
@SMLuthi 👍  thanks! once linting passes this LGTM